boswatch.py

```diff
@@ -202,7 +204,7 @@
 	try:
 		logging.debug("reading config file")
 		globals.config = ConfigParser.ConfigParser()
-		globals.config.read(globals.script_path+"/config/config.ini")
+		globalVars.config.read([globalVars.script_path+"/config/config.ini", "globalVars.script_path+"/config/config_filter.ini"])
```

Comment:
Wieso wurde die Konfig ausgelagert?

Reply:
hehe, hat indirekt auch nichts mit dem Plugin zu tun, habe ich gemacht um die Filter direkt per PHP ändern zu können ohne in die Config selbst reinzuschreiben.. 

Merke: für das nächste mal nicht alles in einen PR ;)